apis/shared-dimensions/lib/handlers/shared-dimension.ts

```diff
@@ -40,7 +41,11 @@ export const put = protectedResource(shaclValidate, asyncMiddleware(async (req,
   return res.dataset(dimension.dataset)
 }))
 
-export const getExport = protectedResource(cors({ exposedHeaders: 'content-disposition' }), asyncMiddleware(async (req, res) => {
+export const getExport = protectedResource(cors({ exposedHeaders: 'content-disposition' }), asyncMiddleware(async (req, res, next) => {
+  if (!req.dataset) {
+    return next(new error.BadRequest())
+  }
```

Comment:
Would be nice to actually create a suggestion as we discussed in rdfjs/express-handler

Reply:
What do you mean?

Reply:
You proposed that it would be better if the `req.dataset` was always there but throw if there is nothing to parse (no body or not RDF)

Reply:
https://github.com/rdfjs-base/express-handler/issues/14